static/bidder-params/greedygame.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "GreedyGame Adapter Params",
+  "description": "A schema which validates params accepted by the GreedyGame adapter",
+  "type": "object",
+
+  "properties": {
+    "publisherId": {
+      "type": ["integer", "string"],
+      "description": "Publisher ID",
+      "minimum": 1,
+      "pattern": "^[1-9][0-9]*$"
+    }
+  },
+
+  "required": ["publisherId"]
```

Comment:
This adapter is an alias of `limelightDigital` adapter. Please use the exact same bidder params as `limelightDigital`.

Reply:
It is fixed in https://github.com/prebid/prebid-server/pull/3083/commits/3114ba42cb13d5c3a953ecf73cec40df31b42a66 commit